chef_master/source/release_notes.rst

```diff
@@ -231,25 +233,25 @@ Changed Flags
      - Notes
    * - ``--[no-]host-key-verify``
      - ``--ssh-verify-host-key VALUE``
-     - See above for valid values. 
+     - See above for valid values.
    * - ``--forward-agent``
      - ``--ssh-forward-agent``
      -
-   * - ``--session-timeout MINUTES`` 
+   * - ``--session-timeout MINUTES``
      - ``--session-timeout SECONDS``
      - New for ssh, existing for winrm. The unit has changed from MINUTES to SECONDS for consistency with other timeouts.
    * - ``--ssh-password``
      - ``--connection-password``
      -
    * - ``--ssh-port``
-     - ``--connection-port`` 
+     - ``--connection-port``
      - ``knife[:ssh_port]`` config setting remains available.
    * - ``--ssh-user``
      - ``--connection-user``
      - ``knife[:ssh_user]`` config setting remains available.
    * - ``--ssl-peer-fingerprint``
      - ``--winrm-ssl-peer-fingerprint``
-     - 
+     -
    * - ``--prerelease``
      - ``--channel CHANNEL``
      - This now allows you to specify the channel that Chef Infra Client gets installed from. Valid values are _stable_,   _current_, and _unstable_. 'current' has the same effect as using the old --prerelease.
```

Comment:
Quick question - do these values need to be italicized or formatted a different way? (See: `_stable_`, `_current_`, and `_unstable_`) I can do this in a follow-up PR.

Reply:
They were in the release notes. It might be nice to have them pop a bit here